### ISSUE TITLE
[fvt]2.12.2 xCAT probe osdeploy miss function "convert_to_epoch_seconds"

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1697]
env:all platform
build:
[root@c910f02c01p14 ~]# lsxcatd -v
Version 2.12.2 (git commit 665bf66f47cf569d4799e0eeb7d078ae1c66f293, built Sat Aug 13 06:15:57 EDT 2016)
[root@c910f02c01p14 ~]# 
How to reproduce:
```
[root@c910f02c01p14 ~]# xcatprobe osdeploy -n c910f02c01p08 -r 1h30m
All pre_defined nodes are valid                                                                                   [ OK ]
The installation network interface is eth1                                                                        [INFO]
Start to search logs after 'Mon Aug 15 00:48:54 2016', please waiting for a while.............
Can't locate object method "convert_to_epoch_seconds" via package "probe_utils" at /opt/xcat/probe//subcmds/osdeploy line 877, <$fd> line 1.
[root@c910f02c01p14 ~]#

```

**Comment by hu-weihua** [comment]
Fixed by pull request #1701, @junxiawang , could you help to verify this bug? thanks

**Comment by junxiawang** [comment]
verified
